modules/react/src/utils/position-children-under-views.js

```diff
@@ -79,7 +79,7 @@ export default function positionChildrenUnderViews({children, viewports, deck, C
           deck._onViewStateChange(params);
         }
       };
-      return createElement(ContextProvider, {value: contextValue}, viewElement);
+      return createElement(ContextProvider, {key: 'context', value: contextValue}, viewElement);
```

Comment:
This key is not unique.

Reply:
- Run `yarn lint fix` to apply prettier.
- You can just use the `key` variable in this function.